tools/utils.py

```diff
@@ -111,7 +111,13 @@ def IsWindows():
 
 
 def SearchFiles(dir, ext):
-  list = glob.glob(dir+ '/**/*.' + ext, recursive=True)
+  list = []
+  files = os.walk(dir)
+  for ent in files:
+    (path, dirs, files) = ent
```

Comment:
```suggestion
  for (path, dirs, files) in os.walk(dir):
```
Is considered Pythonic and is faster, uses less RAM, and `ent` vs. `ext` will not confuse the reader.